packages/apps/desktop-viewer-test/.env

```diff
@@ -1,7 +1,7 @@
 # ---- Configuration ----
 ITWIN_VIEWER_SCOPE ="imodelaccess:read offline_access imodels:read projects:read realitydata:read"
-ITWIN_VIEWER_CLIENT_ID=""
-ITWIN_VIEWER_REDIRECT_URI=""
+ITWIN_VIEWER_CLIENT_ID="native-SH5Fa978k7N8BRLTvNKgtcwRr"
```

Comment:
please dont commit these. create a `.env.local` file and add these there

Reply:
changed